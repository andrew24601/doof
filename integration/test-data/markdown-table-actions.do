// Markdown table action conclusion column (=)
function main(): int {
    let x: int = 3;
    let z: int = 0;

    | x         | = |
    | ---       | --- |
    | 1         | println("one"); |
    | 2, 3      | println("two-or-three"); z = 1; println(z); |
    |           | println("default"); |

    return 0;
}
