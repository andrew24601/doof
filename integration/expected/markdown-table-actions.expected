two-or-three
1
